chroot-bin/croutonwheel

```diff
@@ -46,8 +62,10 @@ if [ $# -ge "$OPTIND" ]; then
     exit 1;
 fi
 
-# Operate on the Chromium OS X11 server
-eval "`host-x11`"
+if [ ! -L /etc/crouton/cmt ]; then
```

Comment:
Can you rebase off of my `xmethods` branch?  This should check if the xmethod is `xorg` and /etc/crouton/cmt is a link (use -h instead of -L).  Although -d might be better, since you do want the mount to exist (in case the target was partially removed for some reason)...